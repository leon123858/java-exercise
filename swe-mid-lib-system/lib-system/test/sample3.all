# 非法與合法 addBook
3
AuthorA SubjectA
AuthorB SubjectB
AuthorC SubjectC
2
Staff UserA
Borrower UserB 2
UserB addBook
AuthorD SubjectD # Error
UserA addBook
AuthorA SubjectA
UserA listSubject SubjectA # ID: 1 Author: AuthorA Subject: SubjectA\nID: 4 Author: AuthorA Subject: SubjectA