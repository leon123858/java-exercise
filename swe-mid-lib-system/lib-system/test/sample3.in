3
AuthorA SubjectA
AuthorB SubjectB
AuthorC SubjectC
2
Staff UserA
Borrower UserB 2
UserB addBook
AuthorD SubjectD
UserA addBook
AuthorA SubjectA
UserA listSubject SubjectA