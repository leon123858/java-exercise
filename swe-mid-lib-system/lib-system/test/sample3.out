Error
ID: 1 Author: AuthorA Subject: SubjectA
ID: 4 Author: AuthorA Subject: SubjectA